File: GLIM-BACKEND/src/main/java/com/lovedbug/geulgwi/core/domain/curation/dto/response/CurationItemResponse.java
package com.lovedbug.geulgwi.core.domain.curation.dto.response;

import lombok.Builder;
import java.util.List;
import com.lovedbug.geulgwi.core.domain.curation.constant.CurationType;

@Builder
public record CurationItemResponse(
    Long curationItemId,
    String title,
    String description,
    CurationType curationType,
    List<CurationContentResponse> contents
) {
}

File: GLIM-AOS/app/src/main/java/com/ssafy/glim/core/data/dto/response/CurationItemResponse.kt
package com.ssafy.glim.core.data.dto.response

data class CurationItemResponse(
    val curationItemId: Long?,
    val title: String,
    val description: String,
    val curationType: String,
    val contents: List<CurationContentResponse>
)

data class CurationContentResponse(
    val bookId: Long?,
    val bookTitle: String,
    val author: String,
    val publisher: String,
    val bookCoverUrl: String?,
    val quoteId: Long?,
    val imageName: String?
)